api/src/test/java/com/mybooks/api/model/MockBook.java

```diff
@@ -0,0 +1,9 @@
+package com.mybooks.api.model;
+
+public class MockBook {
```

Comment:
why we need this class??
If it's for test then it could be defined in a test class or you can create json file in test/resources and load the json file. then parse it to Book class